openapi/paths/customers@{id}@edd-timeline.yaml

```
@@ -41,21 +41,21 @@ post:
     - Core
   tags:
     - Customers
-  summary: Create an EDD Timeline comment
+  summary: Create an EDD timeline comment
   operationId: PostEddTimeline
   x-sdk-operation-name: createEddTimelineComment
-  description: |
-    Create an EDD Timeline comment or custom defined event.
+  description: |-
+    Creates an EDD timeline comment or custom defined event.
```

It is not possible to create custom defined events with this endpoint
```suggestion
    Creates an EDD timeline comment.
```